R/module_teal.R

```diff
@@ -136,6 +138,27 @@ srv_teal <- function(id, modules, teal_data_rv, filter = teal_slices()) {
       title = "SessionInfo"
     )
 
+    if (!(is_mocked(session) || is_in_test())) {
```

Comment:
assuming this is a downloadHandler with the time-consuming processing being a (conditional) handler of a button click - do you still need to put this in the explicit if condition?

Reply:
I'm not sure if I follow. The time-consuming processing is started way faster. We do not start this time-consuming process in mocked shiny session and testing environment, so it is specified here explicility in what conditions it is allowed to be used. If you remove the if condition, then you need to change a bit of how you handle `lockfile_task` later on.

Reply:
Alrighty, got it :) I think you are right. This can be removed. Will try to run R CMD CHECK without it. Maybe there was a reason I put it here that I don't remember anymore.